src/js/index.js

```diff
@@ -14,7 +14,8 @@ const sort = {
 const contentEl = document.getElementById('content');
 const filterInput = document.getElementById('filter');
 const sortingInput = document.getElementById('sorting');
-
+const color_switch = document.getElementById('color-switch');
+const body = document.body;
```

Comment:
This is not needed as `document.body` it's a pretty straightforward reference (vs `getElementById` function).
CMIIW